README.md

```diff
@@ -41,6 +41,12 @@ Experiment<Integer> e = new Experiment("foo");
 e.runAsync(this::controlFunction, this::candidateFunction);
 ```
 
+To run control on the same thread but different for candidate:
```

Comment:
Can you expound on the reasoning behind this a bit more in the README? I think it should be a bit more informative

Reply:
done